tests/platforms/windows/app/test_package.py

```diff
@@ -27,12 +28,50 @@ def package_command(tmp_path):
         version="81.2.1.0",
         arch="groovy",
     )
+    command.tmp_path = tmp_path
     return command
 
 
+@pytest.fixture
+def package_command_with_files(package_command):
+    # Build the path for the source folder:
+    src_path = (
+        package_command.tmp_path
+        / "base_path"
+        / "build"
+        / "first-app"
+        / "windows"
+        / "app"
+        / "src"
+    )
+    src_path.mkdir(parents=True)
+
+    # Mock some typical folders and files in the src folder:
+    paths = (
+        "app/first-app/resources/",
+        "app/first-app-0.0.1.dist-info/",
+        "app_packages/toga_winforms/",
+    )
+    package_command.files = (
+        "First App.exe",
+        "python.exe",
+        "python3.dll",
+        "vcruntime140.dll",
+        "app/first-app/app.py",
+        "app/first-app/resources/__init__.py",
+        "app_packages/clr.py",
+        "app_packages/toga_winforms/command.py",
+    )
+    for path in paths:
+        (src_path / path).mkdir(parents=True)
+    for file in package_command.files:
+        open(f"{src_path}/{file}", "x").close()
```

Comment:
We've got a `create_file()` utility method in `tests/utils` for this purpose - it creates a file with known content, and ensures the directory exists.

Reply:
Used it now.